### DISCUSSION TITLE
Local Error:Solutions cant be run from your current IP Address.

### DISCUSSION THREAD

**Discussion by dr-roshyara** [spatie/laravel-ignition#85] [General]
Dear all, I am getteting everytime the following error, however I can not determine the cause of this error. Could you please suggest me where this error is coming from in Laravel and how can I avoid it. ? 
 I would be thankful if you could help . 
Kindest regards
Nab 
local.ERROR: Solutions cannot be run from your current IP address. {"exception":"[object] (Spatie\\LaravelIgnition\\Exceptions\\CannotExecuteSolutionForNonLocalIp(code: 0): Solutions cannot be run from your current IP address. at /var/www/bernstein/vendor/spatie/laravel-ignition/src/Exceptions/CannotExecuteSolutionForNonLocalIp.php:14)
[stacktrace]
#0 /var/www/bernstein/vendor/spatie/laravel-ignition/src/Http/Controllers/ExecuteSolutionController.php(48): Spatie\\LaravelIgnition\\Exceptions\\CannotExecuteSolutionForNonLocalIp::make()
#1 /var/www/bernstein/vendor/spatie/laravel-ignition/src/Http/Controllers/ExecuteSolutionController.php(21): Spatie\\LaravelIgnition\\Http\\Controllers\\ExecuteSolutionController->ensureLocalRequest()
#2 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(48): Spatie\\LaravelIgnition\\Http\\Controllers\\ExecuteSolutionController->__invoke()
#3 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Routing/Route.php(261): Illuminate\\Routing\\ControllerDispatcher->dispatch()
#4 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Routing/Route.php(204): Illuminate\\Routing\\Route->runController()
#5 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Routing/Router.php(725): Illuminate\\Routing\\Route->run()
#6 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}()
#7 /var/www/bernstein/vendor/spatie/laravel-ignition/src/Http/Middleware/RunnableSolutionsEnabled.php(15): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#8 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Spatie\\LaravelIgnition\\Http\\Middleware\\RunnableSolutionsEnabled->handle()
#9 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#10 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Routing/Router.php(727): Illuminate\\Pipeline\\Pipeline->then()
#11 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Routing/Router.php(702): Illuminate\\Routing\\Router->runRouteWithinStack()
#12 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Routing/Router.php(666): Illuminate\\Routing\\Router->runRoute()
#13 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Routing/Router.php(655): Illuminate\\Routing\\Router->dispatchToRoute()
#14 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(167): Illuminate\\Routing\\Router->dispatch()
#15 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}()
#16 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#17 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#18 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle()
#19 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#20 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#21 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle()
#22 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#23 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle()
#24 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(86): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#25 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle()
#26 /var/www/bernstein/vendor/fruitcake/laravel-cors/src/HandleCors.php(38): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#27 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Fruitcake\\Cors\\HandleCors->handle()
#28 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Http/Middleware/TrustProxies.php(39): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#29 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Http\\Middleware\\TrustProxies->handle()
#30 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#31 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(142): Illuminate\\Pipeline\\Pipeline->then()
#32 /var/www/bernstein/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(111): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter()
#33 /var/www/bernstein/public/index.php(52): Illuminate\\Foundation\\Http\\Kernel->handle()
#34 {main}
"}

**Comment by nekiala** [comment]
I'm getting the same error message. Can someone help please.

**Comment by jovylle** [comment]
I also have this.
